source/guide_ts3audiobot.rst

```diff
@@ -0,0 +1,264 @@
+.. highlight:: console
+
+.. author:: Felix Franz <https://www.felix-franz.com>
+
+.. tag:: web
+
+.. _TS3AudioBot: https://github.com/Splamy/TS3AudioBot
+.. _TeamSpeak3: https://teamspeak.com
+.. _youtube-dl: https://github.com/ytdl-org/youtube-dl
+
+###########
+TS3AudioBot
+###########
+
+.. tag_list::
+
+TS3AudioBot_ is a open-source TeamSpeak3 bot, playing music and much more.
+
+.. warning:: For using the bot, you also need a TeamSpeak3_ server!
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * TeamSpeak3_
+  * :manual:`.htaccess <web-documentroot>`
+  * :manual:`supervisord <daemons-supervisord>`
+..  * :manual:`domains <web-domains>`
+  * :manual:`lang-dotnet`
+
+
+License
+=======
+
+TS3AudioBot_ was written by `Splamy <https://splamy.de/>`_ and licensed under `OSL-3.0 <https://github.com/Splamy/TS3AudioBot#license>`_.
+
+
+Installation
+============
+
+Download release file:
+
+::
+
+ [isabell@stardust ~]$ mkdir ts3audiobot
+ [isabell@stardust ~]$ cd ts3audiobot
+ [isabell@stardust ts3audiobot]$ wget https://github.com/Splamy/TS3AudioBot/releases/latest/download/TS3AudioBot_dotnet_core_3.1.zip
+
+Extract zip archive:
+
+::
+
+ [isabell@stardust ts3audiobot]$ unzip TS3AudioBot_dotnet_core_3.1.zip
+ [isabell@stardust ts3audiobot]$ rm TS3AudioBot_dotnet_core_3.1.zip
+
+Now we will configure the bot.
+To set your TeamSpeak user as bot admin, you unique id is required.
+Go to your TeamSpeak client and open the `Identities` window in the `Tools` menu.
+Copy your `unique id`.
+
+To configure the bot, start it an interactively insert your configuration data (unique TeamSpeak user id, server name, server password, ...).
+After the configuration is done the bot will automatically connect to the TeamSpeak3 server.
+
+::
+
+ [isabell@stardust ts3audiobot]$ dotnet TS3AudioBot.dll
```

Comment:
```suggestion
 :emphasize-lines: 1,10,12,17,19
 
 [isabell@stardust ts3audiobot]$ dotnet TS3AudioBot.dll
 16:10:54.4473| INFO|| [============ TS3AudioBot started =============]
 16:10:54.5198| INFO|| [ Date/Time: Sunday, 14 March 2021 16:10:54
 16:10:54.5302| INFO|| [ Version: 0.11.0/master/c7e44e4d
 16:10:54.5310| INFO|| [ Platform: CentOS Linux 7 (Core) (64bit)
 16:10:54.5315| INFO|| [ Runtime: .NET Core (3.1.13) ServerGC:True GC:SustainedLowLatency
 16:10:54.5343| INFO|| [ Opus: libopus 1.0.2 (x64)
 16:10:54.5346| INFO|| [==============================================]
 16:10:55.2626| WARN|| No permission file found.
 Do you want to set up an admin in the default permission file template? [Y/n] Y
 Please enter an admin uid
 YourAdminUID
 16:11:36.1467| INFO|| Creating new permission file ({"AdminUids":["YourAdminUID"], "OverwriteIfExists":false})
 It seems like there are no bots configured.
 Fill out this quick setup to get started.
 Please enter the ip, domain or nickname (with port; default: 9987) where to connect to:
 1.3.3.7:9987
 Please enter the server password (or leave empty for none):
 SecretTSserverPWifAny
 16:12:41.2601| INFO|| Bot "default" connecting to "1.3.3.7:9987"
 16:12:41.8483| INFO|| Started Webserver on port 58913
```
The interactive mode asks quite a few questions. What do you think about adding sample output like this?